airbyte-integrations/connectors/source-salesforce/source_salesforce/spec.json

```diff
@@ -5,7 +5,7 @@
     "title": "Salesforce Source Spec",
     "type": "object",
     "required": ["client_id", "client_secret", "refresh_token"],
-    "additionalProperties": false,
+    "additionalProperties": true,
```

Comment:
Set it to true because [api_type](url) parameter is removed from the `spec` but it is provided on config files. It's temporary solution not to fail checks.